//python -m grpc_tools.protoc -I=protos --python_out=grpc_compiled/ --grpc_python_out=grpc_compiled/ protos/ner.proto

syntax = "proto3";

//import "nlp_services.proto";


service FinanceNER{
    rpc ExtractEntities(NerRequest) returns (NerResponse) {}
}

message NerRequest{
    string text = 1;
    string metadata = 2;
}

message NerResponse{
    string text = 1;
    string metadata = 2;
    string results = 3;
}

